### PR TITLE
Adding a new bird Coco thekiller

### PR DESCRIPTION
**Pull request by luozizhou**
A trial test. 🐦

### CODE REVIEW

File: params/neils_bird.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.1415
```
Comment (ns-rse) [inline params/neils_bird.R]: Could you choose another unique value here please.

File: params/neils_bird.R [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.1415 
+    
+# e.g. "anas_krystallinus"    
+species.name <- "Coco_thekiller"
```
Comment (ns-rse) [inline params/neils_bird.R]: We've just had Coco thekiller added, could you choose a different bird name please.

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for the contribution @luozizhou 

Unfortunately we've just had a bird added in the file `params/neils_bird.R` and with some of the same parameters, could you please changes these and update your Pull Request.

Thanks,

Neil

File: params/neils_bird.R (hunk 2 above)
Comment (luozizhou) [reply params/neils_bird.R]: Thank you for your comments.

**Review by ns-rse — APPROVED** [review]
Brilliant thanks for fixing that @luozizhou

**Comment by luozizhou** [comment]
Hi @ns-rse, I have changed the parameters.

Thank you for your comments.

**Pull request merged**